internal/controllers/machine/machine_controller_noderef_test.go

```diff
@@ -784,6 +784,115 @@ func TestGetManagedLabels(t *testing.T) {
 	}
 }
 
+func TestGetManagedAnnotations(t *testing.T) {
```

Comment:
Considering this func is implemented in helpers, should this test move there too?
(same for TestGetManagedLabels)

Reply:
Yeah, I think that's perfectly reasonable. I ran out of time to do this last night.

Reply:
Ah yep, I'd not got to this yet :) Am working on it now :)